### ISSUE TITLE
.setpdfwrite operator has been deprecated

### ISSUE THREAD

**Issue by fgilio** [GravityMedia/Ghostscript#6]
Hello!
I'v just received this warning while using Ghostscript 9.50:

```
**** WARNING: The .setpdfwrite operator has been deprecated and will be removed entirely
              in the next release of Ghostscript. The functionality of this operator has
              been reduced to increasing the size of the VM threshold. If you believe you
              have a real need for this then you should replace your call to .setpdfwrite
              with:

                  3000000 setvmthreshold
```

I don't know if there's an specific reason for using that operator 🤔 

PD: Thanks a lot for this useful package

**Comment by schroedan** [comment]
Hi @fgilio 
Thank you for your request. At the moment I do not actively maintain this repo. You could help me out and create a PR for this issue. To keep the functionality backwards compatible I suggest to set `.setpdfwrite` if it was not already set and when the gs version is < 9.50 in https://github.com/GravityMedia/Ghostscript/blob/master/src/Device/PdfWrite.php#L180

**Comment by fgilio** [comment]
Hey @schroedan!
I'd like to give this a go. Been thinking that the only way to know the Ghostscript version is to execute `gs -v` and parse the output, that would mean executing two commands every time. Are you ok with that approach? It feels like too much.
I guess a simpler way would be just to remove it, and maybe we can wait some time before doing it.

**Comment by schroedan** [comment]
Hi @fgilio 
The Ghostscript instance, which is passed as constructor argument [here](https://github.com/GravityMedia/Ghostscript/blob/master/src/Device/PdfWrite.php#L69) already provides a public `Ghostscript::getVersion()` method which returns a (cached) version of Ghostscript. This method already gets called in the constructor of the GS class, so there is no overhead. Please assign this version to a private property of the `PdfWrite` device in the constructor and use it for comparison (see example: [here]( https://github.com/GravityMedia/Ghostscript/blob/master/src/Ghostscript.php#L55)).

**Comment by schroedan** [comment]
Is closed by #10
